binary_like_matrix.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from numpy import array
+
+
+def binary_like_matrix() -> array:
+    """
+    The function creates a matrix of 10x10, in which the elements on the borders will be equal to 1, and inside 0
+    :return: A 10x10 matrix wrapped with 1's and filled with 0's.
+    """
+    matrix = np.ones((10, 10), dtype='int')
+    matrix[1:-1, 1:-1] = 0
```

Comment:
This works great, chekc out numpy.pad for another elegant way of doing this.
https://numpy.org/doc/stable/reference/generated/numpy.pad.html